### PR TITLE
add: notion-app-deb

### PR DESCRIPTION
**Pull request by IOplus**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [ ] Add maintainer to pacscript

### CODE REVIEW

File: packagelist [hunk 1]
```diff
@@ -1,277 +1,278 @@
...
-wine-staging-git
-yabridge
-yad
-yafetch-git
-yandex-browser-beta-deb
-yandex-browser-stable-deb
-youtube-dl-git
-zentile-bin
-zotero
+ adapta-gtk-theme-git
```
Comment (Henryws) [inline packagelist:1]: What??

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packagelist (hunk 1 above)
Comment (IOplus) [reply packagelist:1]: This is what happened when I ran `ls -1 packages > packagelist`, as mentioned on the page [Pacscript 101](https://github.com/pacstall/pacstall/wiki/Pacscript-101), in the "updating the packagelist" section. If you count the lines there's only 1 line added.

Comment (IOplus) [reply packagelist:1]: Oh wait, I didn't see it added a symbol in the beginning of every line, sorry, I'll fix it.

**Review by WRM-42 — CHANGES_REQUESTED** [review]

File: packages/notion-app-deb/notion-app-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name="notion-app-deb"
+pkgname="notion-app"
+repology=("project: ${pkgname}")
+version="2.0.18-1"
+url="https://github.com/notion-enhancer/notion-repackaged/releases/download/v${version}/${pkgname}_${version}_amd64.deb"
+depends="libgtk-3-0 libnotify4 libnss3 libxss1 libxtst6 xdg-utils libatspi2.0-0 libuuid1 libsecret-1-0"
```
Comment (WRM-42) [inline packages/notion-app-deb/notion-app-deb.pacscript:6]: `optdepends` and `depends` are already included in the deb package

File: packages/notion-app-deb/notion-app-deb.pacscript [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+breaks="notion-app-enhanced-deb"
```
Comment (WRM-42) [inline packages/notion-app-deb/notion-app-deb.pacscript:7]:
```suggestion
breaks="${pkgname}-enhanced-deb ${pkgname}-enhanced-bin ${pkgname}-enhanced-app ${pkgname}-bin ${pkgname}-app" 
```

**Review by WRM-42 — CHANGES_REQUESTED** [review]

File: packages/notion-app-deb/notion-app-deb.pacscript (hunk 3 above)
Comment (IOplus) [reply packages/notion-app-deb/notion-app-deb.pacscript]: They don't provide a binary package, so I'll remove them.

**Review by Henryws — APPROVED** [review]

**Pull request merged**